src/coreclr/src/vm/assemblyloadcontext.h

```diff
@@ -15,6 +20,30 @@ class AssemblyLoadContext : public IUnknownCommon<ICLRPrivBinder, IID_ICLRPrivBi
 
     STDMETHOD(GetBinderID)(
         /* [retval][out] */ UINT_PTR* pBinderId);
+
+    NativeImage *LoadNativeImage(Module *componentModule, LPCUTF8 nativeImageName, int nativeImageNameLength);
+
+private:
+    // Multi-thread safe access to the list of composite R2R images
```

Comment:
Nit: This structure does not appear to be multi-thread safe on its own.

Reply:
Removed obsolete comment in 9th commit.